remote_config/src/ios/remote_config_ios.mm

```diff
@@ -491,9 +516,24 @@ static Variant FIRValueToVariant(FIRRemoteConfigValue *in_value, ValueInfo *valu
 }
 
 ConfigUpdateListenerRegistration* RemoteConfigInternal::AddOnConfigUpdateListener(
-      std::function<void(ConfigUpdate&&, RemoteConfigError)>
-          config_update_listener) {
-            return nullptr;
+      LambdaConfigUpdateListener<ConfigUpdate, RemoteConfigError> *config_update_listener) {
```

Comment:
It might be necessary to use __block so that the callback function has a reference to config_update_listener. I'm still a bit fuzzy on how __block works in objective C though, so I'm not entirely sure if it is relevant here.

Once you update the integration test to actually call this method and the callback function it will be a bit more clear if there is any issue with the current logic.

Reply:
From the integration test it looks like it works fine

Reply:
Sounds good, this can stay as-is then